### PR TITLE
Add mycotools

### PR DESCRIPTION
**Pull request by xonq**
Add [Mycotools](https://github.com/xonq/mycotools) to bioconda channel

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by xonq** [comment]
@BiocondaBot please add label

File: recipes/mycotools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('mycotools', max_pin="x") }}
+
+requirements:
+  host:
+    - python >=3.0,<4.0
+    - setuptools
```
Comment (bgruening) [inline recipes/mycotools/meta.yaml:54]: Those pins are not needed. Please remove them.

Comment (xonq) [reply recipes/mycotools/meta.yaml]: This problem has been addressed and the checks are passed now that the Azure pipeline is running again.

File: recipes/mycotools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    - ncbiDwnld = mycotools.ncbiDwnld:cli
+    - ome2name = mycotools.ome2name:cli
+    - predb2mtdb = mycotools.predb2mtdb:cli
+    - s2subs = mycotools.s2subs:cli
+    - update_mtdb = mycotools.update_mtdb:cli
+  noarch: python
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('mycotools', max_pin="x") }}
```
Comment (martin-g) [inline recipes/mycotools/meta.yaml:49]:
```suggestion
    - {{ pin_subpackage('mycotools', max_pin="x.x") }}
```

**Review by martin-g — APPROVED** [review]

**Pull request merged**